components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -88,9 +88,7 @@
 import java.net.*;
 import java.security.KeyStore;
 import java.util.*;
-import java.util.zip.DataFormatException;
-import java.util.zip.Inflater;
-import java.util.zip.InflaterInputStream;
+import java.util.zip.*;
```

Comment:
Fixed in https://github.com/wso2/carbon-identity/pull/399/files#diff-99bf8d528cca774c8ad8936fee3ca73cR113